.github/workflows/e2e-full.yml

```diff
@@ -25,7 +25,9 @@ jobs:
             globalnet: 'globalnet'
         include:
           # This is the oldest K8s version we try to support
-          - k8s_version: '1.17'
+          - k8s_version: '1.19'
```

Comment:
I thought line 18 was the newest?

Reply:
Yes but it's not "rely on GHAs to not run them multiple times" but rather the way GHA is meant to work. You can read it on their manual.

You can add extra parameters this way to an existing job, it won't duplicate it.

In our case we're not adding anything so it just keeps that one job there as is, it won't duplicate it either way.